source/controller/party.js

```diff
@@ -0,0 +1,47 @@
+import data from '../db/storage';
+
+class Party {
+  static getAllParty(req, res) {
+    let parties = data[0];
+    let formattedParties = [];
+    parties.forEach((party) => {
```

Comment:
This is because, in the response specification, the response of this endpoint does not contain one element that is in the parties data. So I am formatting it so that the response will match exactly what's in the specification